python/src/wslink/server.py

```diff
@@ -166,7 +166,8 @@ def create_webserver(server_config, backend="aiohttp"):
 # and start it.
 # =============================================================================
 def start_webserver(
-    options, protocol=wsl.ServerProtocol, disableLogging=False, backend="aiohttp"
+    options, protocol=wsl.ServerProtocol, disableLogging=False, backend="aiohttp",
+    exec_mode="main"
```

Comment:
can you add a **kwargs

Reply:
that way if we pass things we don't care we are still ok

Reply:
Done.